### PR TITLE
python3Packages.prometheus-client: enable tests

### PR DESCRIPTION
**Pull request by fabaff**
###### Motivation for this change
- Package was renamed
- Tests are enabled now

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by fabaff** [comment]
This breaks too many packages for a simple update...

**Comment by fabaff** [comment]
@ofborg eval

File: pkgs/development/python-modules/prometheus-client/default.nix [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+  checkInputs = [
+    pytestCheckHook
+  ];
+
+  pythonImportsCheck = [ "prometheus_client" ];
+
+  meta = with lib; {
+    description = "Prometheus instrumentation library for Python applications";
+    homepage = "https://github.com/prometheus/client_python";
+    license = licenses.asl20;
```
Comment (SuperSandro2000) [inline pkgs/development/python-modules/prometheus-client/default.nix:27]: Missing maintainer.

**Comment by SuperSandro2000** [comment]
There are also other uses of the alias.

File: pkgs/top-level/python-aliases.nix [hunk 2]
```diff
@@ -57,4 +57,5 @@ mapAliases ({
   smmap2 = throw "smmap2 has been deprecated, use smmap instead."; # added 2020-03-14
   topydo = throw "python3Packages.topydo was moved to topydo"; # 2017-09-22
   websocket_client = websocket-client;
+  prometheus_client = prometheus-client; # added 2021-06-10
```
Comment (SuperSandro2000) [inline pkgs/top-level/python-aliases.nix:60]: Please sort this alphabetically.

**Pull request merged**